src/untangled/client/impl/data_fetch.cljs

```diff
@@ -239,9 +243,9 @@
 
       (set-global-loading reconciler))))
 
-(defn- error-callback [reconciler items]
-  (let [loading-items (into #{} (map set-loading! items))]
-    (fn [error]
+(defn- error-callback [reconciler]
+  (fn [error items]
```

Comment:
This changes when `set-loading!` is called and the list of loading-items is generated - not sure if that has side-effects we need to worry about?